cmd/invokeDelete.go

```diff
@@ -0,0 +1,44 @@
+// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+// ------------------------------------------------------------
+
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/dapr/cli/pkg/invoke"
+	"github.com/dapr/cli/pkg/print"
+	"github.com/spf13/cobra"
+)
+
+var invokeResourceID string
+
+// invokeDeleteCmd represents the invokeDelete command
+var invokeDeleteCmd = &cobra.Command{
+	Use:   "invokeDelete",
+	Short: "Issue HTTP DELETE to Dapr app",
+	Run: func(cmd *cobra.Command, args []string) {
+		err := invoke.Delete(invokeAppID, invokeAppMethod, invokeResourceID)
+		if err != nil {
+			print.FailureStatusEvent(os.Stdout, fmt.Sprintf("Error invoking app %s: %s", invokeAppID, err))
+			return
+		}
+
+		print.SuccessStatusEvent(os.Stdout, fmt.Sprintf("HTTP Delete to method %s invoked successfully", invokeAppMethod))
+	},
+}
+
+func init() {
+	invokeDeleteCmd.Flags().StringVarP(&invokeAppID, "app-id", "a", "", "the app id to invoke")
+	invokeDeleteCmd.Flags().StringVarP(&invokeAppMethod, "method", "m", "", "the method to invoke")
+	invokeDeleteCmd.Flags().StringVarP(&invokeResourceID, "id", "", "", "the resource id")
```

Comment:
What is resource id?

Reply:
The ID (key) of the resource to delete. I know Dapr refers to the State ID as Key; however, ID is traditionally used in APIs to designate the target resource.

Reply:
Yeah, but we need to keep consistency and Dapr refers to these as methods..

Reply:
Method is the route, what about the ID of the state to delete?

This was my intention:
http://localhost:PORT/VERSION/invoke/APPID/method/**ID**

For example, if the user wants to delete state with key "99", are you suggesting they should set method as **order/99** and do away with ID?

Reply:
Please see these docs below.  When using service invocation in dapr, only the app-id and the method name are passed in the url.  If there is data to pass it should be in the body.

https://github.com/dapr/docs/blob/master/reference/api/service_invocation.md

https://github.com/dapr/docs/blob/master/howto/invoke-and-discover-services/README.md

Reply:
Sorry, I disagree. Delete REST calls do not have a body. Please answer my question: how do I supply the KEY of the state to be deleted?

Reply:
I am going to close this PR and send a new one with my revisions.